jobs/sql/extractions/hiv_monthly_reporting.sql

```diff
@@ -96,6 +105,56 @@ set latest_hiv_viral_load_coded	= obs_from_group_id_value_coded_list(latest_hiv_
 update temp_eom_appts t
 set latest_hiv_viral_load	= obs_from_group_id_value_numeric(latest_hiv_viral_load_obs_group,'CIEL','856');
 
+-- latest arv treatment line 
+-- a temp table is loaded with the most recent HIV order information for each row
+-- this is used to update the main temp table
+drop temporary table if exists temp_last_arv_order_id;
+create temporary table temp_last_arv_order_id 
+	select t.monthly_reporting_id , o.order_id, o.date_activated, o.encounter_id  
+	from temp_eom_appts t 
+	inner join 
+		(select o2.order_id, o2.patient_id ,  o2.date_activated, o2.encounter_id  
+		from orders o2 
+		where o2.voided = 0
+		and o2.order_reason = concept_from_mapping('CIEL','138405')
+		order by if(o2.date_stopped is null,0,1) asc, o2.date_activated desc) o on o.patient_id = t.patient_id and o.date_activated < t.reporting_date  
+	group by t.monthly_reporting_id 
+;
+
+update temp_eom_appts t 
+inner join temp_last_arv_order_id tlao on tlao.monthly_reporting_id = t.monthly_reporting_id 
+set latest_arv_regimen_order_id = tlao.order_id,
```

Comment:
It doesn't look like this latest_arv_regimen_order_id is used by anything, and doesn't end up in the warehouse.  Why are you populating this?  I think using the order_id that I was concerned about above, just to get the encocunter information and date_activated information is fine, as it doesn't matter that there might be multiple in this case.

Reply:
you are correct... I can remove order_id.  I only care about the encounter_id associated with the order.